virtual_rainforest/models/hydrology/hydrology_model.py

```diff
@@ -669,34 +751,26 @@ def calculate_soil_evaporation(
     barton_ratio = (1.8 * soil_moisture) / (soil_moisture + 0.3)
     alpha = np.where(barton_ratio > 1, 1, barton_ratio)
 
-    saturation_vapour_pressure = DataArray(
-        0.6112 * np.exp((17.67 * (temperature_k)) / (temperature_k + 243.5))
+    saturation_vapour_pressure = 0.6112 * np.exp(
+        (17.67 * (temperature_k)) / (temperature_k + 243.5)
     )
 
-    saturated_specific_humidity = DataArray(
-        (gas_constant_water_vapour / latent_heat_vapourisation)
-        * (
-            saturation_vapour_pressure
-            / (atmospheric_pressure - saturation_vapour_pressure)
-        ),
-        dims=temperature.dims,
+    saturated_specific_humidity = (
+        gas_constant_water_vapour / latent_heat_vapourisation
+    ) * (
+        saturation_vapour_pressure / (atmospheric_pressure - saturation_vapour_pressure)
     )
```

Comment:
I'd split this calculation into a few separate steps because otherwise it's rather difficult to read.

Reply:
Honestly, in the grand scheme of things, this one isn't so bad! I'm not sure alternatives are necessarily clearer.

```python
saturated_specific_humidity =  gas_constant_water_vapour / latent_heat_vapourisation
saturated_specific_humidity *= ( saturation_vapour_pressure / (atmospheric_pressure - saturation_vapour_pressure))

Reply:
Maybe not... up to you, @vgro.